packages/frontend2/src/app/(legacy)/governance/publications/page.tsx

```diff
@@ -0,0 +1,14 @@
+import type { Metadata } from 'next'
+
+export const metadata: Metadata = {
+  title: 'Governance publications - L2BEAT',
+  description:
+    'Explore publications from L2BEAT Governance, discover the latest insights, analyses, and updates on Layer 2 project governance, curated by our L2BEAT Governance Team',
+  openGraph: {
+    url: '/governance/publications',
+  },
+}
+
+export default function Page() {
+  return <>All Publications</>
```

Comment:
If this is incomplete should it be left there?

Reply:
yup, left it just for a second there